hera_qm/ant_metrics.py

```diff
@@ -343,67 +388,77 @@ def __init__(self, dataFileList, reds, fileformat='miriad'):
-                                          self.reds, xants=xants, rawMetric=rawMetric)
+        """Local wrapper for red_corr_cross_pol_metrics in hera_qm.ant_metrics module."""
+        return red_corr_cross_pol_metrics(self.data, self.pols,
+                                          self.antpols, self.ants,
+                                          self.reds, xants=xants,
+                                          rawMetric=rawMetric)
 
     def reset_summary_stats(self):
-        '''Resets all the internal summary statistics back to empty.'''
-
+        """Reset all the internal summary statistics back to empty."""
         self.xants, self.crossedAntsRemoved, self.deadAntsRemoved = [], [], []
         self.removalIter = {}
-        self.allMetrics, self.allModzScores = [], []
+        self.allMetrics, self.allModzScores = OrderedDict(), OrderedDict()
         self.finalMetrics, self.finalModzScores = {}, {}
 
     def find_totally_dead_ants(self):
-        '''Flags antennas whose median autoPower that they are involved in is 0.0.
-        These antennas are marked as dead, but they do not appear in recorded antenna
-        metrics or zscores. Their removal iteration is -1 (i.e. before iterative flagging).'''
+        """Flag antennas whose median autoPower that they are involved in is 0.0.
 
-        autoPowers = compute_median_auto_power_dict(self.data, self.pols, self.reds)
-        power_list_by_ant = {(ant, antpol): [] for ant in self.ants for antpol
-                             in self.antpols if (ant, antpol) not in self.xants}
-        for (ant0, ant1, pol), power in autoPowers.items():
+        These antennas are marked as dead, but they do not appear in recorded antenna
+        metrics or zscores. Their removal iteration is -1 (i.e. before iterative flagging).
+        """
+        autoPowers = compute_median_auto_power_dict(self.data,
+                                                    self.pols,
+                                                    self.reds)
+        power_list_by_ant = {(ant, antpol): []
+                             for ant in self.ants
+                             for antpol in self.antpols
+                             if (ant, antpol) not in self.xants}
+        for ((ant0, ant1, pol), power) in autoPowers.items():
             if (ant0, pol[0]) not in self.xants and (ant1, pol[1]) not in self.xants:
                 power_list_by_ant[(ant0, pol[0])].append(power)
                 power_list_by_ant[(ant1, pol[1])].append(power)
-        for key, val in power_list_by_ant.items():
+        for (key, val) in power_list_by_ant.items():
             if np.median(val) == 0:
                 self.xants.append(key)
                 self.deadAntsRemoved.append(key)
                 self.removalIter[key] = -1
 
-    def _run_all_metrics(self):
-        '''Designed to be run as part of AntennaMetrics.iterative_antenna_metrics_and_flagging().'''
-
+    def _run_all_metrics(self, iter=0):
```

Comment:
I feel like it is easy to just provide flags for each function? run_function=True for all functions? since there are only four function calls under this line